### ISSUE TITLE
Default radiation pattern per element

### ISSUE THREAD

**Issue by Dr-Drive** [zinka/arraytool#2]
Hello again, 
Although I do not think this in an issue, I will make the inquiry through this means anyway. Regarding the radiation pattern per element on the planar and linear arrays, does your code assume an isotropic, dipole or a patch radiator?

If your code assumes an isotropic radiator, which way you think it would be the best to implement an array of arbitrary radiators such as patch or slot antennas?

In other words, using your functions scheme, how would one go about multiplying the array factor function by the  radiation pattern of the arbitrary radiator to obtain the radiation pattern of the array of arbitrary radiators?

**Comment by zinka** [comment]
Hi,

Till now, I haven't included any code for "pattern multiplication". All
elements are assumed to be isotropic only.

One option to do pattern multiplication is to simply use functions such as
"pattern_u, pattern_t, pattern_tp, etc" to calculate array factor
and multiply it with the corresponding 2D/3D element pattern pattern
(either by importing a .csv file or using a simple function to generate
elements patterns for a few ideal elements such as slots, patches, dipoles,
etc.). However, both array factors and elements patterns should be given as
inputs in the "same format".

Hope that helps!

Regards,
zinka

**Comment by Dr-Drive** [comment]
I see, thanks... I have been using the pattern_tp function to extract the Array factor, however, I have not found any pointers regarding the format of the data in your documentation or in the code comments...
Could you please confirm the type of information contained in each AF(i,j) element? Does the "i"  correspond to the intensity in the theta/u axis and the "j", in the phi/v axis? I got to this conclusion by looking at the csv exported data of the tht, phi and AF arrays in excel... Is this correct?

**Comment by zinka** [comment]
I haven't provided any documentation on the "data format" that is generated
by these functions ... but, yes ... what you guessed is the answer ... :)
